tests/integration/test_attach_partition_from_table_with_different_expression/test.py

```diff
@@ -0,0 +1,111 @@
+import pytest
```

Comment:
It seems that regular stateless test will work? Integrations tests are heavy, and it is better to avoid them if possible.

Reply:
At the time I wrote this code, integ tests were the only one I knew how to write lol.

Will move it to stateless.